lib/solver.hpp

```diff
@@ -51,18 +51,18 @@ namespace quda
   /**
      @brief Generate a Krylov space in a given basis
      @param[in] diracm Dirac matrix used to generate the Krylov space
-     @param[out] Ap dirac matrix times the Krylov basis vectors
-     @param[in,out] p Krylov basis vectors; assumes p[0] is in place
+     @param[out] Ap dirac matrix times the Krylov basis vector sets
+     @param[in,out] p Krylov basis vector sest; assumes p[0] is in place
```

Comment:
'test' is misspelled as 'sest' here.

Reply:
[05b2bc6](https://github.com/lattice/quda/pull/1489/commits/05b2bc6e1d1f2012b19b13da9a72fa5cb21f8b00)